### PR TITLE
Feature: Screensaver support enable display videos

### PR DESCRIPTION
**Pull request by tamnguyenchi93**
Show videos is disable by default. This series is also support disable play video sound. But depending on https://github.com/giejay/MediaSliderView/pull/1

### CODE REVIEW

File: app/src/main/java/nl/giejay/android/tv/immich/playback/ScreenSaverService.kt [hunk 1]
```diff
@@ -61,13 +61,26 @@ class ScreenSaverService : DreamService() {
             // first fetch one album, show the (first few) pictures, then fetch other albums and shuffle again
             if (albums.isNotEmpty()) {
                 apiClient!!.listAssetsFromAlbum(albums.first()).map { album ->
-                    val randomAssets = album.assets.shuffled()
+
+                    val randomAssets: List<Asset>;
```
Comment (giejay) [inline app/src/main/java/nl/giejay/android/tv/immich/playback/ScreenSaverService.kt:65]: You can inline this by directly assigning random assets to the outcome of the if else expression

**Review by giejay — COMMENTED** [review]
Thanks for this nice PR! Just one small comment, other than that, looks good!

File: app/src/main/java/nl/giejay/android/tv/immich/playback/ScreenSaverService.kt (hunk 1 above)
Comment (tamnguyenchi93) [reply app/src/main/java/nl/giejay/android/tv/immich/playback/ScreenSaverService.kt:65]: @giejay can you suggest the change? I have no idea what kotlin feature is that.

Comment (giejay) [reply app/src/main/java/nl/giejay/android/tv/immich/playback/ScreenSaverService.kt:65]:
```
val randomAssets: List<Asset> =
                    if (PreferenceManager.screensaverIncludeVideos()) {
                        album.assets.shuffled()
                    } else {
                          album.assets.filter {it.type != "VIDEO"}.shuffled()
                    }
```

Comment (tamnguyenchi93) [reply app/src/main/java/nl/giejay/android/tv/immich/playback/ScreenSaverService.kt:65]: As we already inline assign, does `: List<Asset>` still need?

Comment (giejay) [reply app/src/main/java/nl/giejay/android/tv/immich/playback/ScreenSaverService.kt:65]: No, it's optional

Comment (tamnguyenchi93) [reply app/src/main/java/nl/giejay/android/tv/immich/playback/ScreenSaverService.kt:65]: Okie, I will did a quick test and push new version.

Comment (tamnguyenchi93) [reply app/src/main/java/nl/giejay/android/tv/immich/playback/ScreenSaverService.kt]: I updated the code. Sorry the delay

**Pull request merged**